torch/utils/_pytree.py

```diff
@@ -703,20 +703,35 @@ def __repr__(self, indent: int = 0) -> str:
         children_specs_str: str = ""
         if self.num_children > 0:
             indent += 2
-            children_specs_str += self.children_specs[0].__repr__(indent)
+            children_specs_str += self._children_specs[0].__repr__(indent)
             children_specs_str += "," if self.num_children > 1 else ""
             children_specs_str += ",".join(
                 [
                     "\n" + " " * indent + child.__repr__(indent)
-                    for child in self.children_specs[1:]
+                    for child in self._children_specs[1:]
                 ]
             )
         repr_suffix: str = f"{children_specs_str}])"
         return repr_prefix + repr_suffix
 
+    @property
```

Comment:
Does the typing_extensions deprecated decorator work properties btw?

Reply:
The `@deprecated` decorator works with properties.

Reply:
Updated.